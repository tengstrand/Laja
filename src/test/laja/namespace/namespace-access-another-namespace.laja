#namespace abc

#macro outer(x)
Value = {x}##
#end

#set (aaa = 111)

#import "src/test/laja/namespace/namespace-access-another-namespace-import.laja"

#write "target/test-output/namespace-access-another-namespace.out"
{outer(1)}
## The content of a namespace is stored in a java.util.Map.
Namespace $x = {$x}

{$x.print($abc)}
#end
