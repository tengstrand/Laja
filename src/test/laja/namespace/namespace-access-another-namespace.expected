Value = 1
Namespace $x = {xx=123abc}

Namespace = {aaa=111}
